### PR TITLE
docs: update Typography.Text

### LINKED ISSUE #27090: Typography ellipsis  typescript 定义错误
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/antd-reproduction-template-forked-nbkdd?fontsize=14&hidenavigation=1&theme=dark)

### Steps to reproduce
不能将类型“{ rows: number; expandable: false; }”分配给类型“boolean | undefined”。

### What is expected?
ellipsis: boolean | ellipsis

### What is actually happening?
ellipsis: boolean | undefined

| Environment | Info |
|---|---|
| antd | 4.7.0 |
| React | 17.0.0-rc3 |
| System | Windows 10 |
| Browser | Microsoft Edge  86.0.622.38 |

### PR DESCRIPTION
**Pull request by linvva**
ellipsis only supports boolean

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

close #27090 

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      Update Typography.Text doc     |
| 🇨🇳 Chinese |     更新 Typography.Text 文档      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered components/typography/index.zh-CN.md](https://github.com/Liu-Ya/ant-design/blob/patch-1/components/typography/index.zh-CN.md)

### CODE REVIEW

**Comment by Kermit-Xuan** [comment]
英文的也需要更新

File: components/typography/index.zh-CN.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ cover: https://gw.alipayobjects.com/zos/alicdn/GOM1KQ24O/Typography.svg
 | delete | 添加删除线样式 | boolean | false |  |
 | disabled | 禁用文本 | boolean | false |  |
 | editable | 是否可编辑，为对象时可对编辑进行控制 | boolean \| [editable](#editable) | false | [editable](#editable) |
-| ellipsis | 自动溢出省略，为对象时可设置省略行数、是否可展开、添加后缀等 | boolean \| [ellipsis](#ellipsis) | false | [ellipsis](#ellipsis) |
+| ellipsis | 自动溢出省略 | boolean | false |  |
```
Comment (hengkx) [inline components/typography/index.zh-CN.md:28]: 不是 文档的 问题，是支持对象的

Comment (hengkx) [reply components/typography/index.zh-CN.md:28]: https://ant.design/components/typography-cn/#components-typography-demo-ellipsis

Comment (hengkx) [reply components/typography/index.zh-CN.md:28]: 看错了 Text不支持🤣

Comment (Kermit-Xuan) [reply components/typography/index.zh-CN.md:28]: Text 不支持对象，Paragraph 是支持的。

Comment (Liu-Ya) [reply components/typography/index.zh-CN.md:28]: 看代码写的不支持呀

**Pull request merged**